uber/templates/base.html

```diff
@@ -53,7 +53,7 @@
                 {People: [
                     {Attendees: '../registration/'},
                     {Groups: '../groups/'},
-                    {'All Untaken Shifts': ''},
+                    {'All Untaken Shifts': '../jobs/everywhere'},
```

Comment:
Originally this would only show up if we were in ``AT_THE_CON`` mode.  Then the Anthrocon folks changed it so that it would show up but not be clickable unless we were in ``AT_THE_CON`` mode.  Apparently I broke it so that it would just never show up.  If memory serves, I was testing to make sure that we could still have options that showed up but weren't clickable, and then I must have forgotten to add an ``if`` statement to fill in the link.

This should probably be changed so that instead of ``'../jobs/everywhere'`` it says ``'{% if c.AT_THE_CON %}../jobs/everywhere{% endif %}'`` to match what Anthrocon was doing OR just put the if-statement around the entire line to match what we were doing for M13